README.md

```diff
@@ -24,18 +17,80 @@ Change directory to the package under development. Note: an integration is a spe
 
 ```bash
 cd integrations
-cd package/my-integration
+cd package/my-package
 ```
 
-Run the `help` command and see available actions:
+Run the `help` command and see available commands:
 
 ```bash
 elastic-package help
 ```
 
-## GitHub authorization
+## Commands
+
+`elastic-package` current offers the commands listed below. 
+
+Some commands have a _global context_, meaning that they can be run from anywhere and they will have the 
+same result. Other commands have a _package context_; these must be run from anywhere under a package's
+root folder and they will operate on the contents of that package.
+
+For more details on a specific command, run `elastic-package help <command>`.
+
+### `elastic-package help`
+
+Use this command to get a listing of all commands available under `elastic-package` and a brief
+description of what each command does.
+
+_Context: global_
+
+### `elastic-package version`
+
+Use this command to print the version of `elastic-package` that you have installed. This is
+especially useful when reporting bugs.
+
+_Context: global_
+
+### `elastic-package build`
+
+Use this command to build a package. TODO: what is the purpose of building a package — when would
+a package developer want/need to run this command?
```

Comment:
@mtojek Could you help answer these TODO questions here, please? Thanks!

Reply:
The `build` command copies package internals to the build directory and encode Kibana dashboards. Now it will also skip `_dev` directory.

Reply:
Thanks, I understand what it does internally. I was more wondering why a package developer would want or need to run this command? What use case does it support? Maybe it's supposed to be a pre-requisite for some other commands or something like that?

Reply:
I'm ok with both options, let's wait with announcing them and silently observe in the wild first.

Reply:
Sorry, but I didn't understand. Did you mean to post your answer to a different comment thread?

Reply:
> Sorry, but I didn't understand. Did you mean to post your answer to a different comment thread?

Yes, I had poor network connection. I (or JS) must have messed up sth with anchors. 

Anyway, this is good question. I agree with you that there is no direct goal of running `build` command. Currently built artifacts are used to:
1. Build docker image consisting of package-registry with custom packages.
2. Push changes from integrations to the package-storage.
3. Dry-run package-registry (I believe we should move towards package-spec, as that was an ultimate goal). @ruflin do you know if there are any features that block switching to spec validation?

Reply:
Not now, but in the future I expect this to build the `.zip` file of the package. 

++ on switching to package spec validation and remove the validation code itself from the registry. Each validation part we remove we can check if it is also in the spec.

Reply:
Additional note: We should still have the validation in the package-registry but run it based on package-spec. So far some time we will probably do double validation but this should not be an issue.

Reply:
Thanks for the clarifications @mtojek. 

> 1. Build docker image consisting of package-registry with custom packages.
> 2. Push changes from integrations to the package-storage.

I have added some text covering these two use cases.

> 3. Dry-run package-registry.

I don't understand what this means, how it's different from 1., and how locally-built packages help here, particularly from a package author's perspective. It sounds like this is an internal use, not something package authors need to care about. So I have not mentioned this in the README text for this command.